test/go.mod

```diff
@@ -0,0 +1,17 @@
+module github.com/fingerprintjs/fingerprint-pro-server-api-go-sdk/test
+
+go 1.21
+
+replace github.com/fingerprintjs/fingerprint-pro-server-api-go-sdk/v5 => ../
```

Comment:
this is not needed AFAIK. It is not preventing us from merging the PR, but let's remember to remove it later in a different PR

Reply:
We are actually able to run tests without this because we are using workspaces. I've removed this in [this commit](https://github.com/fingerprintjs/fingerprint-pro-server-api-go-sdk/commit/5c04f6071b4e530b41653d0d626c71dd3f38c070) before